src/arviz_plots/plots/compareplot.py

```diff
@@ -0,0 +1,137 @@
+"""Compare plot code."""
+from importlib import import_module
+
+from arviz_base import rcParams
+
+
+def plot_compare(
+    cmp_df,
+    color="black",
+    similar_band=True,
+    relative_scale=False,
+    figsize=None,
+    target=None,
+    backend=None,
+):
+    r"""Summary plot for model comparison.
+
+    Models are compared based on their expected log pointwise predictive density (ELPD).
+
+    Notes
+    -----
+    The ELPD is estimated either by Pareto smoothed importance sampling leave-one-out
+    cross-validation (LOO) or using the widely applicable information criterion (WAIC).
+    We recommend LOO in line with the work presented by [1]_.
+
+    Parameters
+    ----------
+    comp_df : pandas.DataFrame
+        Result of the :func:`arviz.compare` method.
+    color : str, optional
+        Color for the plot elements. Defaults to "black".
+    similar_band : bool, optional
+        If True, a band is drawn to indicate models with similar
+        predictive performance to the best model. Defaults to True.
+    relative_scale : bool, optional.
+        If True scale the ELPD values relative to the best model.
+        Defaults to False.
+    figsize : (float, float), optional
+        If `None`, size is (10, num of models) inches.
+    target : bokeh figure, matplotlib axes, or plotly figure optional
+    backend : {"bokeh", "matplotlib", "plotly"}
+        Select plotting backend. Defaults to rcParams["plot.backend"].
+
+    Returns
+    -------
+    axes :bokeh figure, matplotlib axes or plotly figure
+
+    See Also
+    --------
+    plot_elpd : Plot pointwise elpd differences between two or more models.
+    compare : Compare models based on PSIS-LOO loo or WAIC waic cross-validation.
+    loo : Compute Pareto-smoothed importance sampling leave-one-out cross-validation (PSIS-LOO-CV).
+    waic : Compute the widely applicable information criterion.
+
+    References
+    ----------
+    .. [1] Vehtari et al. (2016). Practical Bayesian model evaluation using leave-one-out
+       cross-validation and WAIC https://arxiv.org/abs/1507.04544
+    """
+    information_criterion = ["elpd_loo", "elpd_waic"]
+    column_index = [c.lower() for c in cmp_df.columns]
+    for i_c in information_criterion:
+        if i_c in column_index:
+            break
+    else:
+        raise ValueError(
+            "cmp_df must contain one of the following "
+            f"information criterion: {information_criterion}"
+        )
+
+    if backend is None:
+        backend = rcParams["plot.backend"]
+
+    if relative_scale:
+        cmp_df = cmp_df.copy()
+        cmp_df[i_c] = cmp_df[i_c] - cmp_df[i_c].iloc[0]
+
+    if figsize is None:
+        figsize = (10, len(cmp_df))
+
+    p_be = import_module(f"arviz_plots.backend.{backend}")
+    _, target = p_be.create_plotting_grid(1, figsize=figsize)
```

Comment:
```suggestion
    chart, target = p_be.create_plotting_grid(1, figsize=figsize)
    plot_collection = PlotCollection(
        viz=DataTree.from_dict({
            "/": Dataset({"chart": np.array(chart, dtype=object), "plot": np.array(target, dtype=object)}
        })
    )
```

Reply:
Do I need to set data?

Reply:
I don't understand the question. Do you mean storing the artists back into the plot collection? If so no, only the return needs to be plotcollection for the gallery to work (moreover, the target in plotly is kind of a hack, the important object is the chart/figure one so this way both will be returned)

Reply:
I mean that PlotCollection has a data argument. And that argument is mandatory, right?

Reply:
Ooohhh, is it also required via the constructor? Maybe give it an empty dataset if so

Reply:
OK. I will check

Reply:
I passed and empty dataset and I also needed to pass the backend, otherwise I got an error for plotly and bokeh. 

```
File arviz-plots/src/arviz_plots/plot_collection.py:216, in PlotCollection.__init__(self, data, viz_dt, aes_dt, aes, backend, **kwargs)
         self.backend = backend
     elif "chart" in viz_dt:
          self.backend = viz_dt["chart"].item().__module__.split(".")[0]
     if aes is None and aes_dt is not None:
         aes = _get_aes_dict_from_dt(aes_dt)

AttributeError: 'NoneType' object has no attribute '__module__'
```

Reply:
yes, backend is needed too, otherwise the pc.show() part won't work. That was an initial attempt at identifying backend from the provided object, but it doesn't really work as you saw, it does work sometimes, but the automatic backend identification is still pending